src/pages/annual_report/Report.js

```diff
@@ -162,179 +143,177 @@ const Report = function () {
     <div className="reportContainer">
       <MenuBar />
-      <OrganizationUnitModal
-        status={orgUnitModalStatus}
-        changeOrganisationUnitStatus={setOrgUnitModalStatus}
-      />
-      <div className="menu-parent">
-        <div className="menu-parent-container">
-          <div className="data-set-container">
-            <div
-              className="dataset-indication"
-              onClick={() => setGroupVisibility("flex")}
+      <div className="for-selector-bar">
+        <SelectorBar
+          additionalContent={
+            <div className="additional-content">
+              <span>
+                <Button small primary>
+                  Generate report
+                </Button>
+              </span>
+              <span>
+                <Button small>Print</Button>
+              </span>
+            </div>
+          }
+          style={{ width: "800px" }}
+        >
+          <div>
+            <SelectorBarItem
+              label="Group"
+              value={selectedItem}
+              open={_dataGroupVisibility}
+              setOpen={() => _setDataGroupVisibility((prev) => !prev)}
             >
-              <div>Group</div>
-              <div>{selectedItem}</div>
-              <div>
-                <img src={down_allow} />
+              <div
+                className="data-set-info"
+                style={{ width: "100%", display: "block" }}
+              >
+                <ul>
+                  {_settings?.map((element, info) => {
+                    return (
+                      <li
+                        key={element.code}
+                        onClick={(e) => {
+                          e.persist();
+                          setSelectedDataSet(e.target.textContent);
+                        }}
+                      >
+                        {element.name}
+                      </li>
+                    );
+                  })}
+                </ul>
               </div>
-            </div>
-            <div className="data-showable" style={{ display: groupVisibility }}>
-              <ul>
-                {_settings?.map((element, info) => {
-                  return (
-                    <li
-                      key={element.code}
-                      onClick={(e) => {
-                        e.persist();
-                        setSelectedDataSet(e.target.textContent);
-                        setGroupVisibility("none");
-                      }}
-                    >
-                      {element.name}
-                    </li>
-                  );
-                })}
-              </ul>
-            </div>
+            </SelectorBarItem>
           </div>
-          <div className="orgunit-container">
-            <div
-              className="ou-indication"
-              onClick={() => setOrgUnitVisibility("flex")}
+          <div>
+            <SelectorBarItem
+              label="Organisation Unit"
+              value="Organisation Unit"
```

Comment:
SelectorBarItem's value prop must be a string (see the error in the console and the component definition: https://github.com/dhis2/ui/blob/master/components/selector-bar/src/selector-bar-item/selector-bar-item.js#L181. This is also why you are not getting the selection displayed.